libs/spark/stories/card.stories.tsx

```diff
@@ -31,20 +41,30 @@ const MyHeadingImg = styled.img`
   vertical-align: bottom;
 `;
 
+const MyFloatingFavButton = styled(Button)`
+  position: absolute;
+  right: 24px;
+  top: 24px;
+`;
+
 export const CardWithImage = () => (
-  <Card>
-    <CardMedia>
+  <Card style={{ maxWidth: 580 }}>
+    <CardMedia style={{ position: 'relative' }}>
       <MyHeadingImg
         src="https://placekitten.com/300/200"
         alt="Kitten placeholder"
       />
+      <MyFloatingFavButton variant="outlined" size="large">
+        <HeartIconDuotone />
+      </MyFloatingFavButton>
     </CardMedia>
     <CardContent>
-      <Typography gutterBottom variant="h6" component="h2">
-        Card Title
+      <Typography gutterBottom variant="heading-md" color="textOnLight">
```

Comment:
Same as above
```suggestion
      <Typography variant="heading-md" color="textOnLight" style={{ marginBottom: '1rem' }}>
```

Reply:
Might it be better to change the `marginBottom` rule for `Typography[gutter]`?

Reply:
Possibly, it's a bit of a toss up though because a 1rem gutter is desirable on a heading, but probably not on a paragraph or smaller text...

Reply:
Let's change it one-off for now but keep it in mind. If this keeps popping up, we can set a better default.

Reply:
Looking at how Material-UI does it for their docs, they set a top/bottom margin for header variants and avoid `gutterBottom` on them. Spark doesn't have such a specification, but filing a design request would be the way to go for that.